slack-api-client/src/test/java/test_with_remote_apis/audit/ApiTest.java

```diff
@@ -182,4 +184,85 @@ public void getLogs_paginated() throws IOException, AuditApiException {
         }
     }
 
+    @Test
+    public void getLogs_all_actions() throws Exception {
```

Comment:
This test verifies all actions - the coverage depends on the activities in the workspace.